world/world_event_scheduler.cpp

```diff
@@ -57,12 +59,53 @@ void WorldEventScheduler::Process(ZSList *zs_list)
 				if (e.event_type == ServerEvents::EVENT_TYPE_RELOAD_WORLD) {
 					LogScheduler("Sending reload world event [{}]", e.event_data.c_str());
 
-					auto pack = new ServerPacket(ServerOP_ReloadWorld, sizeof(ReloadWorld_Struct));
+					auto pack          = new ServerPacket(ServerOP_ReloadWorld, sizeof(ReloadWorld_Struct));
 					auto *reload_world = (ReloadWorld_Struct *) pack->pBuffer;
 					reload_world->global_repop = ReloadWorld::Repop;
 					zs_list->SendPacket(pack);
 					safe_delete(pack);
 				}
+				if (RuleB(Parcel, EnableParcelMerchants) && RuleB(Parcel, EnablePruning) && e.event_type == ServerEvents::EVENT_TYPE_PARCEL_PRUNE) {
+					LogScheduler("Parcel Prune Event Reached [{}]", e.event_data);
```

Comment:
Should this be a configurable scheduler event type? It seems like this should just run in the world process loop as its own pruner processor

Reply:
See similar to player events truncation, this should live similarly. I don't think this belongs in the server event scheduler

Reply:
I moved the prune routine to a world loop process from the scheduler process.